package.json

```diff
@@ -25,32 +25,30 @@
     "format": "prettier --write \"./**/*.{ts,md}\"",
     "format:check": "prettier --list-different \"./**/*.{ts,md}\"",
     "test": "jest --config jest.config.js --coverage",
-    "lint": "eslint -c .eslintrc --ext .ts src tests",
+    "lint": "eslint -c eslint.config.mjs",
     "build": "tsc"
   },
   "dependencies": {
     "chalk": "^4.1.2",
     "plur": "^4.0.0"
   },
   "peerDependencies": {
-    "eslint": "^8.3.0"
+    "eslint": "^9.0.0"
```

Comment:
Should this be both versions? Otherwise this is technically a breaking change (the code itself didn't change)

Reply:
^8.3.0 || ^9.0.0"?

Reply:
Yeah, that's what I'd do.

Reply:
ok, I've changed peerDependencies and published the new version